libs/dh/admin/feature-permissions/src/lib/overview/dh-admin-permission-overview.component.html

```diff
@@ -0,0 +1,50 @@
+<!--
+@license
+Copyright 2020 Energinet DataHub A/S
+
+Licensed under the Apache License, Version 2.0 (the "License2");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+<watt-card *transloco="let t; read: 'admin.userManagement.permissionsTab'">
+  <watt-card-title>
+    <h4>{{ t("headline", { numberOfPermissions: permissions?.length ?? 0 }) }}</h4>
```

Comment:
suggestion: maybe `dataSource.data.length` can be used, thus avoiding an extra property. 

issue: There's an issue with pluralization - one will still say "1 permissions" but we're most definitely going to have more than one permission, so I don't think this is going to be a problem

Reply:
I didnt do pluralization, on the note, that where will always be more than one :-)